internal/bundle/testdata/failing_lint.mql.yaml

```diff
@@ -10,7 +10,7 @@ policies:
       - checks:
           - uid: sshd-01
             title: Ensure the port is set to 22
-            mql: sshd.config.params["Port"] == 22
```

Comment:
Do we want to keep the old syntax in a test somewhere to make sure it continues to function?

Reply:
Those files are not used for testing. Tests are in https://github.com/mondoohq/cnquery/blob/main/mqlc/mqlc_test.go#L1607-L1639